lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -366,33 +366,45 @@ export async function getReleaseNotesMd(
             .replace(regEx(/^\s*#*\s*/), '')
             .split(' ')
             .filter(Boolean);
-          let body = section.replace(regEx(/.*?\n(-{3,}\n)?/), '').trim();
+          const body = section.replace(regEx(/.*?\n(-{3,}\n)?/), '').trim();
+          const notesSourceUrl = getNotesSourceUrl(
+            baseUrl,
+            repository,
+            project,
+            changelogFile,
+          );
+          const mdHeadingLink = title
+            .filter((word) => !isHttpUrl(word))
+            .join('-')
+            .replace(regEx(/[^A-Za-z0-9-]/g), '');
+          const url = `${notesSourceUrl}#${mdHeadingLink}`;
+          // Look for version in title
           for (const word of title) {
             if (word.includes(version) && !isHttpUrl(word)) {
               logger.trace({ body }, 'Found release notes for v' + version);
-              const notesSourceUrl = getNotesSourceUrl(
-                baseUrl,
-                repository,
-                project,
-                changelogFile,
-              );
-              const mdHeadingLink = title
-                .filter((word) => !isHttpUrl(word))
-                .join('-')
-                .replace(regEx(/[^A-Za-z0-9-]/g), '');
-              const url = `${notesSourceUrl}#${mdHeadingLink}`;
```

Comment:
Lifted these up out of the loop over the title words. Allows them to also be used in the block where we search the body for the version number being released